.github/workflows/ci.yml

```diff
@@ -430,6 +432,7 @@ jobs:
   build_test_ubuntu:
     name: "Local: Build and unit testing on Ubuntu"
     runs-on: ubuntu-latest
+    if: github.ref != 'refs/heads/main'
```

Comment:
```suggestion
    if: github.ref != 'refs/heads/main'
    needs: [docs_build, smoke-tests]
```

Reply:
this has been suggested by @RobPasMue many times. 

Because MAPDL unit tests are flaky, I prefer to have them running asap. So I can evaluate what is going on and rerun if needed. I rather not wait for the documentation and smoke test to finish to run something that I am "kind of" waiting for it.

Reply:
What about decoupling the tests and the build instead of having both in the same job?j

Reply:
Also, do we know the reason behind these spurious failures?

Reply:
it is just our docker image is not very stable.